scripts/style-checker.sh

```diff
@@ -52,12 +52,35 @@ function astyle_c_format
     exit 0
 }
 
+function astyle_c_diff
+{
+    astyle --options="$root_dir/.astylerc" "$root_dir/*.c" |
+    grep "Formatted" |
```

Comment:
I mean lines 58-65. AFAIK the sources are formatted, so those files are changed. This, it is enough to simply call `git diff` after formatting files using `astyle`.